openapi/components/schemas/warframe.yaml

```diff
@@ -1,101 +1,160 @@
 allOf:
   - $ref: ./item.yaml
   - required:
-      - armor
-      - aura
-      - color
-      - conclave
+      - name
+      - uniqueName
       - description
+      - passiveDescription
+      - type
+      - sex
+      - category
+      - productCategory
+      - tradable
+      - isPrime
+      - masterable
+      - vaulted
+      - conclave
+      - wikiAvailable
+      - masteryReq
+      - releaseDate
+      - estimatedVaultDate
+      - vaultDate
+      - imageName
+      - wikiaThumbnail
+      - wikiaUrl
+      - color
+      - sprint
+      - sprintSpeed
+      - stamina
+      - shield
       - health
-      - info
-      - location
-      - mr
-      - name
-      - polarities
+      - armor
       - power
-      - regex
-      - shield
-      - speed
-      - thumbnail
-      - url
+      - aura
+      - polarities
+      - excalted
+      - abilities
+      - pathlogs
+      - components
+      - introduced
+      - bpCost
+      - buildPrice
+      - buildQuantity
+      - consumeOnBuild
+      - buildTime
+      - skipBuildTimePrice
+      - marketCost
     type: object
     properties:
-      shield:
-        minLength: 1
-        type: number
-      polarities:
-        type: array
-        items:
-          type: string
-      prime_power:
+      name:
         minLength: 1
         type: string
-      prime_mr:
+      uniqueName:
         minLength: 1
         type: string
-      color:
-        type: number
-      prime_polarities:
-        type: array
-        items:
-          type: string
-      conclave:
-        minLength: 1
-        type: boolean
       description:
         minLength: 1
         type: string
-      prime_armor:
-        minLength: 1
-        type: number
-      speed:
+      passiveDescription:
         minLength: 1
         type: string
-      aura:
+      type:
         minLength: 1
         type: string
-      prime_url:
+      sex:
         minLength: 1
         type: string
-      prime_health:
-        type: number
-      power:
-        minLength: 1
-        type: number
-      prime_aura:
+      category:
         minLength: 1
         type: string
-      info:
+      productCategory:
         minLength: 1
         type: string
-      thumbnail:
+      tradable:
+        type: boolean
+      isPrime:
+        type: boolean
+      masterable:
+        type: boolean
+      vaulted:
+        type: boolean
+      conclave:
+        type: boolean
+      wikiAvailable:
+        type: boolean
+      masteryReq:
+        minimum: 0
+        type: number
+      releaseDate:
         minLength: 1
         type: string
-      mr:
+      estimatedVaultDate:
```

Comment:
thank you for adding a bunch of these, but if you're going to overhaul it, please sort the keys alphabetically